test/controller/box.js

```diff
@@ -577,11 +577,38 @@ describe('BoxController', function () {
     });
   });
   describe('editing a box', () => {
-    let box;
+    let box, box2;
     beforeEach(async () => {
       const res = await agent.post('/api/v1/box').send({name: 'Pillbox', visibility: 'listed'});
+      const res2 = await agent.post('/api/v1/box').send({name: 'Pillbox2', visibility: 'listed'});
       expect(res.statusCode).to.equal(201);
+      expect(res2.statusCode).to.equal(201);
       box = res.body;
+      box2 = res2.body;
+    });
+    it('updates pokemon _boxVisibility', async () => {
```

Comment:
This looks good, but I think we should also have a similar test for when something is uploaded directly to an unlisted box.